packages/@expo/cli/src/run/ios/appleDevice/AppleDevice.ts

```diff
@@ -79,6 +79,7 @@ function coerceUsbmuxdPlatformToOsType(platform: string): OSType {
   // The only connectable device I have to test against...
   switch (platform) {
     case 'iPhone':
+    case 'iPad':
```

Comment:
This should probably return `iPadOS`

Reply:
Not according to the logs (note the `DeviceClass` type sent by Apple):
```
  expo:apple-device:protocol:lockdown Response: {"Request":"GetValue","Value":{"BoardId":60,"BuildVersion":"20D67","CPUArchitecture":"arm64e","ChipID":32807,"DeviceClass":"iPad","DeviceColor":"1","DeviceName":"Marc-Olivier’s iPad","DieID":5164304050962478,"HardwareModel":"J417AP","HasSiDP":true,"PartitionType":"GUID_partition_scheme","ProductName":"iPhone OS","ProductType":"iPad8,9","ProductVersion":"16.3.1","ProductionSOC":true,"ProtocolVersion":"2","SupportedDeviceFamilies":[1,2],"TelephonyCapability":false,"UniqueChipID":5164304050962478,"UniqueDeviceID":"00008027-001258E83C78402E","WiFiAddress":"64:0b:d7:b1:4b:2d"}} +31ms
```

Reply:
I think he was talking about the return statement that state ios instead of ipad os. I will need to take a look further in the implementation to see if this change something. 

Ps: @EvanBacon can you remove my ipad serial number from your copied log, i edited my PR description to remove it on my side.

Reply:
Ok, i have taken a look at it and ios seems to be the good parameter. It is also listed in Xcode as ios: 
![image](https://github.com/user-attachments/assets/f4221bb2-c249-41d5-9509-fc411bbbf5bf)